src/EditorFeatures/Test2/Expansion/NameExpansionTests.vb

```diff
@@ -222,7 +222,7 @@ class C
             Await TestAsync(input, expected, expandParameter:=True)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Expansion)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/11979"), Trait(Traits.Feature, Traits.Features.Expansion)>
```

Comment:
The failure (in this context) is indeed a consequence of this change, but the same failure occurred before when the context is changed slightly (#11979). I don't have enough context to attempt a fix.

---
In reply to: [67016763](https://github.com/dotnet/roslyn/pull/11981#discussion_r67016763) [](ancestors = 67016763,67015720)